packages/react/src/components/MenuItem/MenuItem.stories.mdx

```diff
@@ -10,7 +11,13 @@ export const meta = {
 
 <Meta title={meta.title} component={meta.component} />
 
-export const Template = args => <MenuItem {...args} />;
+export const Template = (args) => {
+    return (
+        <Menu open={true} role="menu">
+            <MenuItem {...args} />
+        </Menu>
+    )
+};
```

Comment:
I guess we are using 4 spaces here

Reply:
FIxed with https://github.com/wso2/oxygen-ui/pull/21/commits/59365a5711b8bf4b43fb857ef2b95c8446611b76